tidy3d/plugins/adjoint/components/data/monitor_data.py

```diff
@@ -167,8 +167,8 @@ def package_flux_results(self, flux_values: JaxDataArray) -> float:
         """How to package the dictionary of fields computed via self.colocate()."""
         flux_data = flux_values
         if isinstance(flux_data, JaxDataArray):
-            return jnp.sum(flux_data.values)
-        return jnp.sum(flux_data)
+            return jnp.sum(jnp.nan_to_num(flux_data.values))
+        return jnp.sum(jnp.nan_to_num(flux_data))
```

Comment:
still needed?

Reply:
no but maybe doesn't hurt. what do you think?

Reply:
I don't know, I feel like nan can alert us about something wrong, while converting to 0-s would produce wrong values and be harder to catch.